File: app/src/main/java/io/plan8/backoffice/view/Plan8TaskCalendarView.java
package io.plan8.backoffice.view;

import android.content.Context;
import android.util.AttributeSet;

import com.prolificinteractive.materialcalendarview.MaterialCalendarView;

/**
 * Created by chokwanghwan on 2017. 11. 28..
 */

public class Plan8TaskCalendarView extends MaterialCalendarView {
    private boolean isAlreadyInflated = false;

    public Plan8TaskCalendarView(Context context) {
        super(context);
    }

    public Plan8TaskCalendarView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }


    public boolean isAlreadyInflated() {
        return isAlreadyInflated;
    }

    public void setAlreadyInflated(boolean alreadyInflated) {
        isAlreadyInflated = alreadyInflated;
    }
}

File: app/src/main/java/io/plan8/backoffice/view/Plan8TaskCalendarView.kt
package io.plan8.backoffice.view

import android.content.Context
import android.util.AttributeSet
import android.view.MotionEvent
import com.prolificinteractive.materialcalendarview.MaterialCalendarView

/**
 * Created by chokwanghwan on 2017. 11. 6..
 */
class Plan8TaskCalendarView : MaterialCalendarView {
    constructor(context: Context) : super(context) {}

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {}

    var isAlreadyInflated: Boolean = false
}